src/libaudqt/audqt.cc

```diff
@@ -126,6 +153,26 @@ EXPORT QVBoxLayout * make_vbox (QWidget * parent, int spacing)
     return layout;
 }
 
+EXPORT QFont get_font_for_class (const char *className)
+{
+    QString currentStyle = qapp->style ()->objectName();
+    if (currentStyle.compare ("Macintosh", Qt::CaseInsensitive) == 0
+        || currentStyle.contains ("Aqua", Qt::CaseInsensitive))
+    {
+        static QHash<const QString, const char *> map = {
+            { "QListView", "QSmallFont" },
+            { "QListBox", "QSmallFont" },
+        };
+        if (map.contains (className))
+        {
+            QFont ret = QApplication::font (map[className]);
+            qDebug() << Q_FUNC_INFO << className << "->" << map[className] << "->" << ret.toString();
+            return ret;
+        }
+    }
+    return QApplication::font (className);
+}
```

Comment:
Qt already has support for setting fonts on a per-class basis.  Try:
QApplication::setFont (QApplication::font ("QSmallFont"), "QTreeView");

It needs to be QTreeView though, not QListView (I checked and we don't use QListView anywhere).